homeassistant/components/mqtt/__init__.py

```diff
@@ -181,9 +179,6 @@ def embedded_broker_deprecated(value):
                     CONF_CLIENT_CERT, "client_key_auth", msg=CLIENT_KEY_AUTH_MSG
                 ): cv.isfile,
                 vol.Optional(CONF_TLS_INSECURE): cv.boolean,
-                vol.Optional(CONF_TLS_VERSION, default=DEFAULT_TLS_PROTOCOL): vol.Any(
```

Comment:
Can we mark it as `cv.deprecated` so we don't break configs

Reply:
mark it as `cv.deprecated` but drop the code which sets TLS version or mark it as `cv.deprecated` and ignore the parameter?

Reply:
ignore parameter